README.md

```diff
@@ -1,23 +1,28 @@
 # detray
 
-Detray is part of the ACTS project (R&D line for parallelization), the ACTS project can be found: https://github.com/acts-project/acts.
+[![Test Status](https://github.com/acts-project/detray/actions/workflows/builds.yml/badge.svg?branch=main)](https://github.com/acts-project/detray/actions/workflows/builds.yml)
+[![Lint Status](https://github.com/acts-project/detray/actions/workflows/checks.yml/badge.svg?branch=main)](https://github.com/acts-project/detray/actions/workflows/checks.yml)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=acts-project_detray&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=acts-project_detray)
 
-This is a C++20 header only library for detector surface intersections using different algebra plugin libraries. It follows the navigation and propagation concept of ACTS, however, with an attempt to create
-a geometry without polymorphic inheritance structure.
+Detray is part of the [ACTS project](https://github.com/acts-project/acts) (R&D line for parallelization).
 
+This is a C++20 header-only library providing a GPU-friendly tracking detector description using different [linear algebra](https://github.com/acts-project/algebra-plugins) libraries. It follows the navigation and propagation concept of ACTS, however, implementing a geometry using a flat memory layout and no abstract interfaces (virtual functions). A detray detector can therefore be constructed on the host and copied to an accelerator device in a straight-forward way.
 
-## Requirements and dependencies
-#### OS & compilers:
+With the geometry description comes a fully featured, GPU-ready track state propagation implementation in inhomogeneous magnetic fields (vector field description using [covfie](https://github.com/acts-project/covfie)), with track parameter covariance transport including material interactions.
+
+## Requirements and Dependencies
+#### OS & Compilers:
 
 - The C++ compiler must support C++20
 - The CUDA Toolkit version must be greater than major version 11
 
-#### Dependency
-- CMake
+#### Dependencies:
+- CMake (version >= 3.11)
```

Comment:
Note: Some parts of Detray require at least CMake 3.18.

Reply:
Nvm, it seems algebra-plugins needs 3.14 for some reason, so detray has no chance of working with 3.11 actually